### DISCUSSION TITLE
Name of Current Campaign in UI ?

### DISCUSSION THREAD

**Discussion by GrEaSeLiTeNiN** [Khopa/dcs_liberation#1811] [Q&A] [answered]
Hi, sometimes I may forget which campaign name I am currently playing and I cannot find the name of the campaign anywhere in the UI. It would be good if the UI could indicate, somewhere, the name of the current campaign. Or did I miss something? 

Thanks!

**Comment by Starfire13** [comment, marked as answer]
Why not include the name of the campaign in the save file? I always do that.

**Reply by GrEaSeLiTeNiN** [reply]
I guess this is the only way to identify the current campaign name for now if you forget which one it was. Hopefully the next Lib version will include the Campaign name in the UI. 
![image](https://user-images.githubusercontent.com/94265170/143678175-56b18540-1b0c-41a9-9d26-7deb866532fe.png)
